webview-ui/src/components/ResolutionsPage/ModifiedFileMessage.tsx

```
@@ -0,0 +1,955 @@
+        } catch (error) {
+          // Fallback to plain text if highlighting fails
+          highlightedContent = content;
+        }
+      }
+      
+      return (
+        <div key={index} className={`diff-line ${lineClass}`}>
+          <span className="diff-line-number">{lineNumber}</span>
+          <span 
+            className="diff-content"
+            dangerouslySetInnerHTML={{ __html: highlightedContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)